CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -81,6 +81,15 @@ protected function getTemplateUpdates() {
           ['name' => 'event_online_receipt', 'type' => 'html'],
         ]
       ],
+      [
+        'version' => '5.4.alpha2',
```

Comment:
i think this should be 5.5.alpha1 i think

Reply:
yep - if it is against master - see above for why I think 5.4 rc makes sense